upgrade/breaking-changes/4-0-0.md

```diff
@@ -215,11 +215,17 @@ await TreeListRef.SetStateAsync(desiredState);
 
 - The Form columns can have different widths
 
+### Gantt 
+
+- Renamed the `<GanttToolBar>` to `<GanttToolBarTemplate>`
```

Comment:
"The" is used if `<GanttToolBar>` is used in a phrase, e.g. "renamed the <...> tag".

```suggestion
- Renamed `<GanttToolBar>` to `<GanttToolBarTemplate>`.
```